src/Approvals/SFA.DAS.Approvals.Api/Controllers/OverlappingTrainingDateRequestController.cs

```diff
@@ -59,5 +59,17 @@ public async Task<IActionResult> ValidateUlnOverlapOnStartDate(long providerId,
 
             return Ok(result);
         }
+
+        [HttpGet]
+        [Route("{apprenticeshipId}/getOverlapRequest")]
```

Comment:
route still uses apprenticeshipId instead of draftApprenticeshipId

Reply:
Apologies for overlook. that has been corrected now.